### PR TITLE
Include an example for listing all Lambdas in any region

### PR DESCRIPTION
**Pull request by theputta**
##### SUMMARY
To list out all lambdas in any region 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
amazon.aws.lambda_info module 

```paste below

```

### CODE REVIEW

**Review by abikouo — APPROVED** [review]

File: plugins/modules/lambda_info.py [hunk 1]
```diff
@@ -48,6 +48,17 @@
...
+- name: List AWS Lambda Functions
+  tasks:
+    - name: List Lambda Functions
+      amazon.aws.lambda_info:
+        region: us-east-1
+      register: lambda_functions
+    - name: Display Lambda Functions
+      debug:
+        var: lambda_functions.functions  
+
```
Comment (tremble) [inline plugins/modules/lambda_info.py:61]:
```suggestion
# List AWS Lambda Functions in a specific region 
- name: List Lambda Functions
  amazon.aws.lambda_info:
    region: us-east-1
  register: lambda_functions
- name: Display Lambda Functions
  debug:
    var: lambda_functions.functions  
```

**Review by tremble — CHANGES_REQUESTED** [review]
Thanks for taking the time to submit this PR.  Our examples are generally tasks rather than plays

**Comment by theputta** [comment]
> Thanks for taking the time to submit this PR. Our examples are generally tasks rather than plays

@tremble I have updated as per your recommendation

File: plugins/modules/lambda_info.py [hunk 2]
```diff
@@ -48,6 +48,14 @@
...
   register: my_function_details
 
+# List AWS Lambda Functions in a specific region 
+- name: List Lambda Functions
+  amazon.aws.lambda_info:
+    region: us-east-1
+  register: lambda_functions
+- name: Display Lambda Functions
+  debug:
+    var: lambda_functions.functions
```
[the remark this answers is not in the document]
Comment (tremble) [reply plugins/modules/lambda_info.py:58]: region is applicable to almost all of our modules, we generally don't include it in examples (same as we don't include the multitude of mechanisms for passing credentials)

Generally we don't include a debug task, the examples aren't there to demo the debug builtin, and the output info has its own section of the documentation.

**Review by tremble — CHANGES_REQUESTED** [review]

**Review by tremble — APPROVED** [review]

File: plugins/modules/lambda_info.py [hunk 3]
```diff
@@ -48,6 +48,12 @@
     function_name: myFunction
   register: my_function_details
 
+
```
Comment (tremble) [inline plugins/modules/lambda_info.py:51]:
```suggestion
```

**Review by tremble — APPROVED** [review]

**Pull request merged**